net/xl2tpd/patches/100-makefile_opt_flags.patch

```diff
@@ -1,12 +0,0 @@
---- a/Makefile
-+++ b/Makefile
-@@ -97,7 +97,8 @@ OSFLAGS+= -DUSE_KERNEL
- 
- IPFLAGS?= -DIP_ALLOCATION
- 
--CFLAGS+= $(DFLAGS) -Os -fno-builtin -Wall -DSANITY $(OSFLAGS) $(IPFLAGS)
-+OFLAGS=-Os
-+CFLAGS+= $(DFLAGS) $(OFLAGS) -fno-builtin -Wall -DSANITY $(OSFLAGS) $(IPFLAGS)
```

Comment:
Please also remove the overridden of `OFLAGS` in the package Makefile.

Reply:
removed